### PR TITLE
Falkland Islands (Legislative Assembly): refresh data

### PR DESCRIPTION
**Pull request by everypoliticianbot**
Automated data refresh for Falkland Islands - Legislative Assembly

#### Output

```
Add memberships from sources/morph/wikipedia.csv
Merging with sources/morph/official.csv
Merging with sources/morph/wikidata.csv
Adding GenderBalance results from sources/gender-balance/results.csv
  ⚥ data for 75; 36 added

No dates for election Q15744904 (Next Falkland Islands general election)
Creating 10 term files

Top identifiers:
  20 x wikidata
  1 x clara
  1 x viaf

Creating names.csv
Creating unstable/positions.csv
Persons matched to Wikidata: 20 ✓ | 30 ✘
  No wikidata: Harold Rowlands (f0c31a11-1ad4-4d7a-8bf6-eca28b529d4a)
  No wikidata: John Edward Cheek (c636cd6f-fb28-4606-8891-a2ebd02b08dc)
  No wikidata: Terence John Peck (5d427660-db2d-4b3f-86f0-314a99cb64ad)
  No wikidata: Stephen Luxton (d8de53b7-0798-455a-bba0-4e77b61d2017)
  No wikidata: Kevin Kilmartin (3269f76c-cf87-4e6e-859d-122f91fb9132)
  No wikidata: Stuart Barrett Wallace (8de4e6a8-bd09-43a8-a979-520678f2366c)
  No wikidata: Lewis Clifton (81ab01d9-2345-4262-84aa-00014c29bda4)
  No wikidata: Lionel Geoffrey Blake (63fae433-6182-4ff2-a9cb-3846b8b8a57f)
  No wikidata: Ron Binnie (9cf9611e-8877-4f1f-a613-1912bac64424)
  No wikidata: William Henry Goss (72a1c546-2e4d-4919-9669-6bcb59fcd5e2)
Parties matched to Wikidata: 1 ✓ 

```

### CODE REVIEW

**Comment by ondenman** [comment]
Closes: https://github.com/everypolitician/everypolitician-data/issues/16432

**Comment by tmtmtmtm** [comment]
This Pull Request has been superseded by #16738 but there are non-bot commits.

@everypoliticianbot, @ondenman is this pull request still needed?

File: data/Falkland_Islands/Assembly/sources/idmap/wikipedia.csv [hunk 1]
```diff
@@ -48,3 +48,4 @@ william_henry,b6193b32-bc3c-494a-a097-a99c9b544850
 william_edward_bowles,4375aaac-99ee-4bf4-8f88-8937d3feb291
 derek_stanley_evans,1a85276d-912d-4d61-b1c8-5c243578aace
 stuart_barrett_wallace,8de4e6a8-bd09-43a8-a979-520678f2366c
+nonpartisan,98a42dbc-898b-41c0-8f01-7c34088ae186
```
Comment (chrismytton) [inline data/Falkland_Islands/Assembly/sources/idmap/wikipedia.csv]: I think this line can be deleted now we've got rid of the nonpartisan entry?

**Comment by chrismytton** [comment]
@ondenman One tiny comment, once that's fixed this is good to go! 👍  🚀

**Pull request merged**